.drone.yml

```diff
@@ -26,7 +26,7 @@ steps:
     filename: unit-tests-cache.tar
 
 - name: unit-tests
-  image: wjordan/code-dot-org:2020-09-10
+  image: cdodeploy/drone:latest
```

Comment:
Could we name our Docker user (organization?) `code-dot-org`?
Also, our Docker image will hopefully be used in the future in other ways than just to run Drone continuous integration builds. Could we name the image itself `code-dot-org as well?

Reply:
we can't use hyphens in the username, but i can change to `codedotorg`. i will update the image name.